### ISSUE TITLE
Verse Block - Spacing Ignored When Published

### ISSUE THREAD

**Issue by donalirl** [WordPress/gutenberg#18857]
**Describe the bug**
As I understand it, the Verse Block is supposed to leave all spaces untouched, displaying them exactly as you enter them. But, this does not seem to be happening. I have reproduced this on many themes including Twenty Nineteen and Twenty Twenty.

**To reproduce**
1. Open a new page or post and add a verse block
2. Type a poem with spaces inside the text at various points between words (example below)
3. Publish the page

_Example of a poem with spacing:_

```
 This is my poem,
     My lovely lovely poem
I wrote my poem

     it is written
          published
,                              spacing is lost 
```

**Expected behavior**
I expected the spaces I typed in the editor to be preserved on the live site.

**Screenshots**
In the editor:
<img width="594" alt="verse editor" src="https://user-images.githubusercontent.com/30727666/69925157-d4a62e80-1502-11ea-8d12-45c4d6e1b987.PNG">

On the front-end:
<img width="304" alt="verse front end" src="https://user-images.githubusercontent.com/30727666/69925160-da037900-1502-11ea-9335-89ce41907beb.PNG">

Applying this CSS restores the intended spacing:

```
/* Verse Block: add the proper spacings */
pre.wp-block-verse {
    white-space: pre;
}
```

**Desktop (please complete the following information):**
 - OS: Windows
 - Browser: Chrome
 - Version 78

**Additional context**
Using latest Gutenberg version

**Comment by KokkieH** [comment]
I think this might be a duplicate of #19083, which should be fixed via #19173

**Comment by draganescu** [comment]
This seems to be fixed in master now, so I will close this. Most likely as @KokkieH resolved above.
